readme.md

```diff
@@ -69,6 +69,8 @@ Watch task helps you reduce your efforts during development extensions. If the t
 gulp watch
 ```
 
+In order for Live-reload to work, reload extension manually after starting `gulp watch`.
```

Comment:
To  summarize: 

You need to load/reload application **after** starting `gulp watch` for live-reload to work.

For content scripts you need to reload pages where it is used.